packages/data-models/workflows/misc.workflows.ts

```diff
@@ -27,6 +27,51 @@ const workflows: IDeploymentWorkflows = {
       },
     ],
   },
+  // Generate Android assets from source images (splash.png, icon.png and, optionally, icon-foreground.png and icon-background.png)
+  // Icon images must be at least 1024×1024px, splash image must be at least 2732×2732px
+  // Further specifications provided here: https://www.npmjs.com/package/cordova-res
+  populate_android_assets: {
```

Comment:
The workflows all look good to me, although I think probably they are substantial enough that we can move them to their own `android.workflows.ts` file